src/OpenFeature/OpenFeature.cs

```diff
@@ -1,30 +1,34 @@
 ﻿using System;
 using System.Collections.Generic;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Logging.Abstractions;
 using OpenFeature.Model;
 
 namespace OpenFeature
 {
     public sealed class OpenFeature
     {
-        private EvaluationContext _evaluationContext;
+        private EvaluationContext _evaluationContext = new EvaluationContext();
         private IFeatureProvider _featureProvider = new NoOpFeatureProvider();
         private readonly List<IHook> _hooks = new List<IHook>();
-        //TODO Add ILogger interface so we use the standard logging interface
-        
+        public static ILogger Logger { get; private set; } = new Logger<OpenFeature>(new NullLoggerFactory());

```

Comment:
Relevant for later: https://github.com/open-feature/spec/issues/39

Reply:
Yea I was just playing around here, I can move it down to the client level like suggested in open-feature/spec#39

Reply:
Ya, I dont think you should worry about the logger too much now, I just wanted to give you the heads-up.